### ISSUE TITLE
TFTP/SAMBA tab overrides settings in nework tab

### ISSUE THREAD

**Issue by ilia-kats** [tobimensch/aqemu#19]
Currently, folders specified in the Network->TFTP/SAMBA tab simply get appended as extra qemu arguments, overriding any settings specified in Network->Network->smb or tftp. If I understand the purpose of the TFTP/SAMBA tab correctly, it's supposed to simply expose those options more visibly. The correct behavior, IMHO, would then be to synchronize the values of Network->Network->smb and Network->TFTP/SAMBA->SMB (and tftp) in the GUI, and only build QEMU arguments from the Network->Network tab.

**Comment by tobimensch** [comment]
Yeah, that makes sense. The reason for the odd behaviour are two different sets of  command line options (one of which can even be considered deprecated) of qemu, that got reflected in aqemu.
